atomate2/openmm/flows/core.py

```diff
@@ -0,0 +1,51 @@
+
+from atomate2.openmm.jobs.core import InputMaker, EnergyMinimizationMaker, NPTMaker, AnnealMaker, NVTMaker
+from typing import List, Union, Optional
+from pathlib import Path
+
+from dataclasses import dataclass
+
+from pydantic import Field
+from jobflow import Maker, Flow
+
+from pymatgen.io.openmm.schema import InputMoleculeSpec
+
+
+
+@dataclass
+class ProductionMaker(Maker):
+    """
+
+
+    """
+    name: str = "production"
+
+    input_maker: InputMaker = Field(default_factory=InputMaker)
+    energy_maker: EnergyMinimizationMaker = Field(default_factory=EnergyMinimizationMaker)
+    npt_maker: NPTMaker = Field(default_factory=NPTMaker)
+    anneal_maker: AnnealMaker = Field(default_factory=AnnealMaker)
+    nvt_maker: NVTMaker = Field(default_factory=NVTMaker)
+
+    def make(
+        self,
+        input_mol_dicts: List[Union[Dict, InputMoleculeSpec]],
+        density: Optional[float] = None,
+        box: Optional[List[float]] = None,
+        prev_dir: str | Path | None = None
+    ):
+        input_job = self.input_maker.make(input_mol_dicts=input_mol_dicts)
+
+        energy_job = self.energy_maker.make(input_set=input_job.output.input_set)
+
+        pressure_job = self.npt_maker.make(input_set=energy_job.output.input_set)
+
+        anneal_job = self.anneal_maker.make(input_set=pressure_job.output.input_set)
+
+        production_job = self.nvt_maker.make(input_set=anneal_job.output.input_set)
+
+        my_flow = Flow(
+            [input_job, energy_job, pressure_job, anneal_job, production_job],
```

Comment:
What's the significance of the order of the Jobs here? I think I know what input_job and energy_job are doing, but why do you pressure_job --> anneal_jon --> production_job in that order?